numpy/lib/index_tricks.py

```diff
@@ -596,6 +597,419 @@ def __iter__(self):
     next = __next__
 
 
+class ndrange(_nx.collections_abc.Sequence):
+    """
+    The ndrange type represents an immutable sequence of numbers and can be
+    used for looping over specific multi-dimensional indicies in ``for`` loops.
```

Comment:
The first sentence should be shorter because it is quoted elsewhere.

Something like "An N-dimensional range-like object for indexing arrays."

Reply:
Please see https://docs.python.org/3/library/stdtypes.html#ranges

I didn't know range was so core to python before I started this.

They make it seem like it is on the order of importance as a `list` and a `tuple`. 

As in the comment below,  I feel like this is a stylistic change with respect to how numpy/python decide to write their docs. I fully support going toward numpy wording.